src/components/ImagePreviewer.tsx

```diff
@@ -0,0 +1,253 @@
+import React, { useRef, MouseEvent } from "react";
+import styled from "styled-components";
+
+const StyledWrapper = styled.div`
+    position: relative;
+    width: 355px;
+    user-select: none;
+
+    > img {
+        width: 100%;
+        height: auto;
+        pointer-events: none;
+    }
+`;
+
+export interface Rect {
+    x: number;
+    y: number;
+    width: number;
+    height: number;
+    isCompleted: boolean;
+    _id: string;
```

Comment:
Any specific reason to use `_id` rather than `id` here?

Reply:
I set it to `_id` because I sometimes work on the document from mongo DB.
I will change it to `id`.